base/compiler/typeinfer.jl

```diff
@@ -551,9 +551,9 @@ function finish(me::InferenceState, interp::AbstractInterpreter)
         # annotate fulltree with type information,
         # either because we are the outermost code, or we might use this later
         doopt = (me.cached || me.parent !== nothing)
-        recompute_cfg = type_annotate!(interp, me, doopt)
+        type_annotate!(interp, me, doopt)
```

Comment:
We can eliminate anything related to `any_unreachable` within `type_annotate!` now.